packages/superset-ui-demo/.storybook/main.js

```diff
@@ -20,6 +24,17 @@ module.exports = {
     '../storybook/stories/**/*Stories.[tj]sx',
   ],
   webpackFinal: config => {
+
+    // Make sure babel is applied to the package src
+    // These are excluded by the default rule
+    // because they reside in node_modules
+    config.module.rules.push({
+      include: PLUGIN_PACKAGES_PATH_REGEXP,
+      exclude: /node_modules/,
+      test: /\.jsx?$/,
+      use: config.module.rules[0].use,
+    });
+
     config.module.rules.push({
```

Comment:
This rule only applies to typescript. but there are older packages that are jsx.